tests/modules/nf-core/metaphlan/metaphlan/main.nf

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env nextflow
+
+nextflow.enable.dsl = 2
+
+include { UNTAR                 } from '../../../../../modules/nf-core/untar/main.nf'
```

Comment:
```suggestion
include { UNTAR               } from '../../../../../modules/nf-core/untar/main.nf'
```

Reply:
UNTAR was used to extract compressed database tar.gz file. So I think it's necessary to include it here.

Reply:
This was a format suggestion, not removing the whole line :)

Reply:
Sorry. I didn't check carefully. Thanks Sofia!